src/ui_nbgl.c

```diff
@@ -347,6 +366,16 @@ static void confirmation_status_callback(void)
 	if (uiContext.confirmedStatus) {
 		nbgl_useCaseStatus(uiContext.confirmedStatus, true, ui_idle_flow);
 		nbgl_reset_transaction_full_context();
+    } else if(uiContext.standardStatus) {
```

Comment:
Lint is very weird and hard to read

Reply:
I know, but don't want to format the whole app it will be a pain for future PRs from the developer.

It uses real tabs instead of spaces...